Wikipedia/Code/AboutViewController.m

```diff
@@ -59,6 +59,15 @@ - (void)wmf_setTextFontSize {
     [self evaluateJavaScript:fontSizeJS completionHandler:nil];
 }
 
+- (void)wmf_setTextFontColor:(WMFTheme *)theme {
+    NSString *fontColorJS = [NSString stringWithFormat:@"document.body.style.color = '#%@';\
+                             document.styleSheets[0].rules[2].style.color = '#%@';\
```

Comment:
@montehurd if you add an id to `style`, it won't let you access its css rules. It looks like it's useful in different cases: https://stackoverflow.com/questions/3797221/why-would-you-give-a-style-tag-an-id/3797236